### PR TITLE
feat(slatetohtml): convert line breaks to br tags

### PR DESCRIPTION
**Pull request by steven-prybylynskyi**
Addresses https://github.com/thompsonsj/slate-serializers/issues/32

### CODE REVIEW

**Comment by steven-prybylynskyi** [comment]
Although I'll have to add more test cases, to cover nested structures, e.g. within tables or nested div's, etc.

File: __tests__/serializers/slateToHtml/index.spec.ts [hunk 1]
```diff
@@ -122,3 +122,111 @@ describe('empty content', () => {
...
+        children: [
+          {
+            text: 'Paragraph 2',
+          },
+        ],
+      },
+    ]
+    const html = 'Paragraph 1<br>Paragraph 2'
+    expect(slateToHtml(slate, { ...slateToDomConfig, convertLineBreakToBr: true })).toEqual(html)
+  })
```
Comment (thompsonsj) [inline __tests__/serializers/slateToHtml/index.spec.ts:171]: I've given this some thought - seems like the most logical way to deal with this scenario. Thanks for adding it! I can see it involves some additional logic in the serializer.

File: src/serializers/blocks.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
+// See: <https://html.spec.whatwg.org/#the-css-user-agent-style-sheet-and-presentational-hints>
+// From: https://www.npmjs.com/package/rehype-minify-whitespace
+const blocks = [
```
Comment (thompsonsj) [inline src/serializers/blocks.ts:3]: I really appreciate the attention to detail moving this to an appropriate location following it's usage in `slateToDom` as well as `slatetoHtml`.

**Review by thompsonsj — APPROVED** [review]

**Comment by thompsonsj** [comment]
Thanks @steven-prybylynskyi ! Adding support for line breaks in `slateToDom`/`slateToHtml` is something that I hadn't gotten around to yet, so I appreciate your adding support for that.

Agreed with the point about adding tests for nested structures such as tables, nested divs ...etc. I plan to look into that at some point.

Merging this as it is a much needed improvement.

**Comment by thompsonsj** [comment]
Included in [0.0.25](https://github.com/thompsonsj/slate-serializers/releases/tag/v0.0.25)

**Comment by steven-prybylynskyi** [comment]
Thx @thompsonsj! It's really great to see your quick response. That's one of the concerns usually when it comes to do contributions to small libraries and 3rd party libs in general.

Bumping the version here

**Pull request merged**